backend/lib/entities/state/attributes/StatusStateAttribute.js

```diff
@@ -37,7 +37,8 @@ StatusStateAttribute.VALUE = Object.freeze({
     CLEANING: "cleaning",
     PAUSED: "paused",
     MANUAL_CONTROL: "manual_control",
-    MOVING: "moving"
+    MOVING: "moving",
+    POWER_SAVE: "power_save"
```

Comment:
This is not a minor bugfix 👀

Reply:
Debatable.

Reply:
So just map it to idle? That would be fine for me too. Docked is definitely wrong though.